azure-mgmt-batch/src/main/java/com/microsoft/azure/management/batch/BatchAccount.java

```diff
@@ -254,5 +299,34 @@
              */
             Update withoutApplication(String applicationId);
         }
+
+        /**
+         * The stage of a Batch account definition allowing the creation of a Batch pool.
+         */
+        interface WithPool {
+            /**
+             * Starts a definition of an pool to be created in the Batch account.
+             *
+             * @param poolId the reference name for the application
+             * @return the first stage of a Batch pool definition
+             */
+            Pool.UpdateDefinitionStages.Blank<Update> defineNewPool(String poolId);
```

Comment:
it is poolId, not sure whey the `reference name` here?

Reply:
The applicationId has comment with "the reference name for the application". I need to replace application by pool here. But for id/name, should we follow applicationId or change both this time?

Reply:
Then keep the name here.